packages/connect-react/src/components/RemoteOptionsContainer.tsx

```
@@ -0,0 +1,142 @@
+    propName: prop.name,
+    configuredProps: configuredPropsUpTo,
+    dynamicPropsId: dynamicProps?.id,
+  };
+  if (prop.useQuery) {
+    componentConfigureInput.query = query || ""; // TODO ref.value ? Is this still supported?
+  }
+  // exclude dynamicPropsId from the key since only affect it should have is to add / remove props but prop by name should not change!
+  const queryKeyInput = {
+    ...componentConfigureInput,
+  }
+  delete queryKeyInput.dynamicPropsId
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)